packages/mobile/package.json

```diff
@@ -26,6 +26,7 @@
     },
     "dependencies": {
         "@capacitor-community/barcode-scanner": "^2.0.1",
+        "@capacitor-community/privacy-screen": "^2.1.1",
```

Comment:
Can you run `yarn` again so the lockfile is updated?

Reply:
Also this might need to be added to the Podfile and Podfile.lock

Reply:
yes will do now